skbio/metadata/__init__.py

```diff
@@ -16,4 +16,12 @@
 # ----------------------------------------------------------------------------
 
 from skbio.util import TestRunner
+
+from ._interval import IntervalMetadata
```

Comment:
Can you also expose `Interval` (the python class, not the cython class) here, add it to `__all__`, and include it in the docstring above? Even though it's unlikely to be constructed directly by users, it's still a user-facing object so it'd be nice to have it included in the docs and importable from a public location.